### PR TITLE
🌐 [i18n-KO] Translated  `audio_classification.mdx` to Korean

### PR DESCRIPTION
**Pull request by gabrielwithappy**
# What does this PR do?

Translated the `audio_classification.mdx` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team PseudoLab, may you please review this PR? @0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [x] Did you write any new necessary tests?

### CODE REVIEW

**Comment by gabrielwithappy** [comment]
Team PseudoLab, may you please review this PR? @0525hhgus, @kihoon71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

File: docs/source/ko/_toctree.yml [hunk 1]
```diff
@@ -49,8 +49,8 @@
     title: 자연어처리
     isExpanded: false
   - sections:
-      - local: in_translation
-        title: (번역중) Audio classification
+      - local: tasks/audio_classification
+        title: 오디오 분류
       - local: tasks/asr
         title: 자동 음성 인식
     title: (번역중) 오디오
```
Comment (wonhyeongseo) [inline docs/source/ko/_toctree.yml:56]:
```suggestion
     title: 오디오
```

이 섹션은 완료되었으므로 번역중을 빼셔도 될 것 같아요. 좋은 번역 감사합니다, @gabrielwithappy 님!

**Review by wonhyeongseo — CHANGES_REQUESTED** [review]
Section is complete, tysm @gabrielwithappy !

**Comment by gabrielwithappy** [comment]
@wonhyeongseo 
I updated your suggestion!
Thank you for your feedback. :-)

File: docs/source/ko/tasks/audio_classification.md [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+
+>>> num_labels = len(id2label)
+>>> model = AutoModelForAudioClassification.from_pretrained(
+...     "facebook/wav2vec2-base", num_labels=num_labels, label2id=label2id, id2label=id2label
+... )
+```
+
+이제 세 단계만 남았습니다:
+
+1. 훈련 하이퍼파라미터를 [`TrainingArguments`]에 정의합니다. 유일한 필수 파라미터는 모델을 저장할 위치를 지정하는 `output_dir`입니다. `push_to_hub = True`를 설정하여 이 모델을 허브로 푸시합니다(모델을 업로드하려면 허깅 페이스에 로그인해야 합니다). 각 에포크가 끝날 때마다 [`Trainer`]가 정확도를 평가하고 트레이닝 체크포인트를 저장합니다.
```
Comment (0525hhgus) [inline docs/source/ko/tasks/audio_classification.md:216]:
```suggestion
1. 훈련 하이퍼파라미터를 [`TrainingArguments`]에 정의합니다. 유일한 필수 매개변수는 모델을 저장할 위치를 지정하는 `output_dir`입니다. `push_to_hub = True`를 설정하여 이 모델을 허브로 푸시합니다(모델을 업로드하려면 허깅 페이스에 로그인해야 합니다). 각 에폭이 끝날 때마다 [`Trainer`]가 정확도를 평가하고 훈련 체크포인트를 저장합니다.
```

File: docs/source/ko/tasks/audio_classification.md [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+2. 모델, 데이터 세트, 토큰화기, 데이터 콜레이터, `compute_metrics` 함수와 함께 훈련 인자를 [`Trainer`]에 전달합니다.
```
Comment (0525hhgus) [inline docs/source/ko/tasks/audio_classification.md:217]:
```suggestion
2. 모델, 데이터 세트, 토크나이저, 데이터 콜레이터, `compute_metrics` 함수와 함께 훈련 인자를 [`Trainer`]에 전달합니다.
```

File: docs/source/ko/tasks/audio_classification.md [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+...     train_dataset=encoded_minds["train"],
+...     eval_dataset=encoded_minds["test"],
+...     tokenizer=feature_extractor,
+...     compute_metrics=compute_metrics,
+... )
+
+>>> trainer.train()
+```
+
+트레이닝이 완료되면 모든 사람이 모델을 사용할 수 있도록 [`~transformers.Trainer.push_to_hub`] 메서드를 사용하여 모델을 허브에 공유하세요:
```
Comment (0525hhgus) [inline docs/source/ko/tasks/audio_classification.md:250]:
```suggestion
훈련이 완료되면 모든 사람이 모델을 사용할 수 있도록 [`~transformers.Trainer.push_to_hub`] 메소드를 사용하여 모델을 허브에 공유하세요:
```

File: docs/source/ko/tasks/audio_classification.md [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+
+For a more in-depth example of how to finetune a model for audio classification, take a look at the corresponding [PyTorch notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/audio_classification.ipynb).
+
+</Tip>
+
+## 추론[[inference]]
+
+이제 모델을 세밀하게 조정했으니 추론에 사용할 수 있습니다!
+
+추론을 실행할 오디오 파일을 로드합니다. 필요한 경우 오디오 파일의 샘플링 속도를 모델의 샘플링 속도와 일치하도록 리샘플링하는 것을 잊지 마세요!
```
Comment (0525hhgus) [inline docs/source/ko/tasks/audio_classification.md:268]:
```suggestion
이제 모델을 미세 조정했으니 추론에 사용할 수 있습니다!

추론을 실행할 오디오 파일을 가져옵니다. 필요한 경우 오디오 파일의 샘플링 속도를 모델의 샘플링 속도와 일치하도록 리샘플링하는 것을 잊지 마세요!
```

File: docs/source/ko/tasks/audio_classification.md [hunk 6]
```diff
@@ -0,0 +1,329 @@
...
+    {'score': 0.07667109370231628, 'label': 'pay_bill'},
+    {'score': 0.0755252093076706, 'label': 'balance'}
+]
+```
+
+원하는 경우 `pipeline`의 결과를 수동으로 복제할 수도 있습니다:
+
+<frameworkcontent>
+<pt>
+특징 추출기를 로드하여 오디오 파일을 전처리하고 '입력'을 PyTorch 텐서로 반환합니다:
```
Comment (0525hhgus) [inline docs/source/ko/tasks/audio_classification.md:299]:
```suggestion
특징 추출기를 가져와서 오디오 파일을 전처리하고 `입력`을 PyTorch 텐서로 반환합니다:
```

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
좋은 번역 감사합니다 😀
오디오 관련 용어를 꼼꼼하게 번역해주셔서 수월하게 이해했습니다 👍
아래 리뷰를 남기니, 참고 부탁드립니다!

**Comment by gabrielwithappy** [comment]
> 좋은 번역 감사합니다 😀
> 오디오 관련 용어를 꼼꼼하게 번역해주셔서 수월하게 이해했습니다 👍
> 아래 리뷰를 남기니, 참고 부탁드립니다!

꼼꼼한 리뷰 너무감사 드립니다!!🤗

**Review by gabrielwithappy — COMMENTED** [review]
update review

**Comment by gabrielwithappy** [comment]
@sgugger, @ArthurZucker, @eunseojo 
May you please review this PR?

**Comment by LysandreJik** [comment]
cc @stevhliu for review :)

**Review by stevhliu — APPROVED** [review]
Thanks for your translation! 😄

**Pull request merged**